bioio_lif/reader.py

```diff
@@ -1,36 +1,59 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from typing import Any, Optional, Tuple
-
-from bioio_base.dimensions import Dimensions
-from bioio_base.reader import Reader as BaseReader
+import xml.etree.ElementTree as ET
+from copy import copy
+from math import floor
+from typing import Any, Dict, Hashable, List, Optional, Tuple, Union
 
+import dask.array as da
+import numpy as np
 import xarray as xr
+from bioio_base import constants, exceptions
```

Comment:
This 3 `bioio_base` imports could be combined

Reply:
resolved in [c2d764a](https://github.com/bioio-devs/bioio-lif/pull/3/commits/c2d764abf04c346e342f56f798a1f32b5f20539c)